### ISSUE TITLE
Solidified metal in a crucible displays 2400 degrees baking temperature instead of whatever the metal is

### ISSUE THREAD

**Issue by McFrugal** [anegostudios/VintageStory-Issues#4360]
### Game Version

1.19.8

### Platform

Windows

### Modded

Modded

### SP/MP

Singleplayer

### Description

If you have solidified metal in a crucible it will display a baking temperature of 2400 degrees, and no melting temperature displayed for whatever metal is in it.  The baking temperature is useless since it's already baked, and it would be far more useful to show the melting temperature.  It almost gives the impression you need to heat the crucible to 2400 degrees to melt the metal!

### How to reproduce

Have any solidified metal in a crucible, then mouse over it.

### Logs

```Log```

**Comment by Craluminum2413** [comment]
I believe it is needed to allow to reheat crucible, otherwise it wouldn't be possible to reheat without that "2400" thing in tooltip

**Comment by RedramVS** [comment]
The 2400 is the 'smelting' temperature of the crucible itself.  To best of my knowledge this is a code issue, not something I can fix via json.

**Comment by McFrugal** [comment]
> I have the same issue in version 1.20. I tried to melt tin, but for some reason, the metal solidified in the crucible, and now it won’t melt. Tin has a melting point of 232°C, and I heated the crucible up to 1300°C, but nothing happens.

Are you sure?  When you heat solidified metal in a crucible the crucible never goes into the output slot for some reason, but if you mouse over it you can tell when it's melted.

My issue is just a display problem.

**Comment by vane-666** [comment]
> Are you sure? When you heat solidified metal in a crucible the crucible never goes into the output slot for some reason, but if you mouse over it you can tell when it's melted.

It doesn’t show the melting point; instead, it shows the baking temperature of 2400°C.
I suspect that the metal isn’t melting because the system assumes its melting point is 2400°C, rather than the actual value, which should be different.

**Comment by McFrugal** [comment]
> > Are you sure? When you heat solidified metal in a crucible the crucible never goes into the output slot for some reason, but if you mouse over it you can tell when it's melted.
> 
> It doesn’t show the melting point; instead, it shows the baking temperature of 2400°C. I suspect that the metal isn’t melting because the system assumes its melting point is 2400°C, rather than the actual value, which should be different.

Yes, that's what my issue is about, and it's a display problem.  The real melting point is the same melting point as whatever material you have in the crucible.  Like I said, please check to see if it's melted by mousing over it.

**Comment by tannyl** [comment]
Also note that it shows what it "bakes into".

Using any of these "bake" texts does not make sense on the crucible, since clay items are "fired", and this item is already done.

Also these text strings are used things like bread and pie. Using the same text on so different items can make translation difficult. 

![Screenshot from 2024-11-27 18-07-21](https://github.com/user-attachments/assets/2e98273f-2507-483e-af08-8c0fd283323b)

![Screenshot from 2024-11-27 18-07-28](https://github.com/user-attachments/assets/5d4d1566-f875-4278-a2c6-89e0210915ee)

**Comment by RetoranPetra** [comment]
Had this issue, I do kind of expect the temperature to read as what temperature you need to bring the crucible up to to melt the material as opposed to the temperature at which the crucible "bakes" which I assume is just a placeholder temperature that's impossible to reach which is getting displayed in error?

Either way, haven't had an issue with reheating the crucible once I found out the tooltip changes to "melted X" when it's no longer solid, but I spent an embarrassing length of time waiting for it to move from the left to the right, but this doesn't happen when the crucible's metal has been remelted.

I would suggest
- Making it so the crucible remains heated if the firepit is still going even if it's in the output slot
- Making it so that when reheating a crucible it moves to the output slot when it has reached an appropriate temperature
- Hiding the existing "bakes at 2400C" tooltip, and changing it to "melts at X C" or leaving it blank

**Comment by MKMoose** [comment]
> The 2400 is the 'smelting' temperature of the crucible itself. To best of my knowledge this is a code issue, not something I can fix via json.

The cooking pot doesn't have this problem. Just get rid of the `combustibleProps` (at `survival/blocktypes/clay/fired/crucible.json:61-70`) and add an `allowHeating: true` attribute.
